setup.py

```diff
@@ -67,9 +67,6 @@ def run(self):
         # higher.
         # Python 3.9 and later include zoneinfo which replaces pytz
         'pytz>=2015.7; python_version<"3.9"',
-        # https://github.com/python/cpython/issues/95299
-        # https://github.com/python-babel/babel/issues/1031
-        'setuptools; python_version>="3.12"',
```

Comment:
I'm not sure but shouldn't this become an optional dependency? Removing it entirely means the user needs to provision it separately. Unless that is what is intended, but then it should be in the docs somewhere

Reply:
People who are using `setup.py` (who would thus use the `setuptools_frontend` bits) will necessarily be depending on `setuptools`, so I don't think that's absolutely necessary :)